src/coreclr/jit/lower.cpp

```diff
@@ -5447,6 +5449,60 @@ GenTree* Lowering::LowerAdd(GenTreeOp* node)
             JITDUMP("Remove [%06u], [%06u]\n", op2->gtTreeID, node->gtTreeID);
             return next;
         }
+#ifdef TARGET_ARM64
+        if (BlockRange().TryGetUse(node, &use))
```

Comment:
Add some comments here about the purpose of the following code?

Reply:
I can do that, though this kind of code is pretty common in lowering.